Content.Server/Salvage/SalvageSystem.Expeditions.cs

```diff
@@ -148,8 +149,10 @@ private SalvageExpeditionConsoleState GetState(SalvageExpeditionDataComponent co
         return new SalvageExpeditionConsoleState(component.NextOffer, component.Claimed, component.Cooldown, component.ActiveMission, missions);
     }
 
-    private void SpawnMission(SalvageMissionParams missionParams, EntityUid station)
+    private SpawnSalvageMissionJob SpawnMission(SalvageMissionParams missionParams, EntityUid station)
     {
+        var ftlDestination = _entManager.CreateEntityUninitialized("FTLPoint", new EntityCoordinates());
+        
```

Comment:
I don't really understand why this was moved out, or why it spawns at one spawn then gets shifted to another spot at some time after.

Reply:
The reason for this change was because of the async nature of creating maps; since the Salvage map creation process is started when the mission is selected in the console _and_ that's when the CD is simultaneously instantiated, the CD needs to have an empty FTL destination attached to it that later gets overwritten by the correct destination. If this isn't done, the CD simply has null as the FTL destination.

This could cause issues if the map creation process is delayed for long enough that someone manages to print the CD and slot it into a shuttle console before the true FTL destination is set. Afaik the fix would be to simply remove and reinsert the CD, but if this scenario ever happens I would guess something has gone wrong with the map generation code elsewhere.

Reply:
This has been reverted as it's no longer necessary.